internal/e2e/e2e_rank_test.go

```diff
@@ -42,7 +42,6 @@ func TestRanking(t *testing.T) {
 	archiveURLs := []string{
 		"https://github.com/sourcegraph/sourcegraph-public-snapshot/tree/v5.2.2", // Nov 1 2023
 		"https://github.com/golang/go/tree/go1.21.4",                             // Nov 7 2023
-		"https://github.com/sourcegraph/cody/tree/vscode-v0.14.5",                // Nov 8 2023
```

Comment:
You could also switch to https://github.com/sourcegraph/cody-public-snapshot but it's probably not important enough.